contracts/plugins/assets/origin/OETHCollateralL2Base.sol

```diff
@@ -0,0 +1,87 @@
+// SPDX-License-Identifier: BlueOak-1.0.0
+pragma solidity 0.8.19;
+
+import "@openzeppelin/contracts/utils/math/Math.sol";
+import "../../../libraries/Fixed.sol";
+import "../ERC4626FiatCollateral.sol";
+import "../OracleLib.sol";
+
+interface IMorphoChainlinkOracleV2 {
+    function price() external view returns (uint256);
+}
+
+/**
+ * @title Origin Staked ETH Collateral for Base L2
+ * @notice Collateral plugin for Origin OETH,
+ * tok = wsuperOETHb  (wrapped superOETHb)
+ * ref = superOETHb (pegged to ETH 1:1)
+ * tar = ETH
+ * UoA = USD
+ */
+contract OETHCollateralL2Base is ERC4626FiatCollateral {
+    using OracleLib for AggregatorV3Interface;
+    using FixLib for uint192;
+
+    IMorphoChainlinkOracleV2 public immutable targetPerTokChainlinkFeed; // {tar/token}
+
+    AggregatorV3Interface public immutable uoaPerTargetChainlinkFeed; // {UoA/tar}
```

Comment:
why not use the default chainlinkFeed for this one? I think this is what we did for RETH for example.

Reply:
no reason in particular, just didn't want to refactor the submitted code. I agree it's dissimilar to how we've done it in other cases but it seems fine